reference/OpenAPI.json

```diff
@@ -381,6 +381,1013 @@
+          "type": "object",
+          "properties": {
+            "date": {
+              "type": "string"
+            },
+            "related_object_type": {
+              "type": "string"
+            },
+            "related_object_id": {
+              "type": "string"
+            },
+            "related_object_parent_id": {
+              "type": "string"
+            },
+            "stacked": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "SimpleProductDiscountUnit": {
+        "type": "object",
+        "title": "Simple Product Discount Unit",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Unique product ID, assigned by Voucherify."
+          },
+          "source_id": {
+            "type": "string",
+            "description": "Product's source ID."
+          },
+          "name": {
+            "type": "string",
+            "description": "Product name."
+          }
+        },
+        "required": [
+          "id",
+          "name"
+        ]
+      },
+      "SimpleSkuDiscountUnit": {
+        "type": "object",
+        "title": "Simple Sku Discount Unit",
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "Unique SKU ID, assigned by Voucherify."
+          },
+          "source_id": {
+            "type": "string",
+            "description": "Product variant's source ID."
+          },
+          "name": {
+            "type": "string",
+            "description": "Sku name"
+          }
+        },
+        "required": [
+          "id",
+          "name"
+        ]
+      },
+      "DiscountUnit": {
+        "type": "object",
+        "title": "Unit",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "UNIT"
+            ],
+            "description": "Discount type."
+          },
+          "unit_off": {
+            "type": "number",
+            "description": "Number of units to be granted a full value discount."
+          },
+          "unit_off_formula": {
+            "type": "string"
+          },
+          "effect": {
+            "$ref": "#/components/schemas/DiscountUnitVouchersEffectTypes",
+            "description": "Defines how the unit is added to the customer's order."
+          },
+          "unit_type": {
+            "type": "string",
+            "description": "The product deemed as free, chosen from product inventory (e.g. time, items)."
+          },
+          "product": {
+            "$ref": "#/components/schemas/SimpleProductDiscountUnit",
+            "description": "Contains information about the product."
+          },
+          "sku": {
+            "$ref": "#/components/schemas/SimpleSkuDiscountUnit"
+          }
+        }
+      },
+      "DiscountUnitMultiple": {
```

Comment:
<img width="1570" alt="image" src="https://github.com/voucherifyio/voucherify-openapi/assets/117282008/e12438aa-5bcf-48d9-b0ac-255295866262">
- `type` was not returned here (inside the object in array)
- please verify if the `unit_type` is required
<img width="400" alt="image" src="https://github.com/voucherifyio/voucherify-openapi/assets/117282008/72dd0ae0-89d5-4b91-93aa-40a8cc65753c">
it looks like a duplicate - in `UNIT` we have `Simple SKU Discount Unit Object`

Reply:
you are right, i messed up :)